include/eve/arch/arm/as_register.hpp

```diff
@@ -37,8 +36,11 @@ namespace eve
       }
       else if constexpr( std::is_same_v<T,double> && Size::value <= 1 )
       {
-        if constexpr(spy::supports::aarch64_) return float64x1_t{};
-        else                                  return emulated_{};
+        #if defined(SPY_SIMD_IS_ARM_ASIMD)
+        return float64x1_t{};
```

Comment:
What about 64 bit integers? Are they available?

Reply:
like uint64_t ? Yes. Only thing we don't have is SIMD register of 64 bits integer